cba_settings_userconfig/cba_settings.sqf

```diff
@@ -323,14 +323,14 @@ force force ace_repair_engineerSetting_repair = 1;
 force force ace_repair_engineerSetting_wheel = 0;
 force force ace_repair_fullRepairLocation = 3;
 force force ace_repair_fullRepairRequiredItems = ["ace_repair_anyToolKit"];
-force force ace_repair_locationsBoostTraining = false;
+force force ace_repair_locationsBoostTraining = true;
 force force ace_repair_miscRepairRequiredItems = ["ace_repair_anyToolKit"];
 force force ace_repair_repairDamageThreshold = 0.6;
 force force ace_repair_repairDamageThreshold_engineer = 0.4;
 force force ace_repair_wheelRepairRequiredItems = [];
 
 // ACE Magazine Repack
-ace_magazinerepack_repackLoadedMagazines = true;
+force force ace_magazinerepack_repackLoadedMagazines = true;
```

Comment:
Same here. I would actually be against this change personally, I have it intentionally set to `false`, so repacking doesn't empty my weapon, so I'm always ready to shoot.